src/docfx/config/ContributionConfig.cs

```diff
@@ -23,13 +23,15 @@ internal class ContributionConfig
         public readonly string Branch = string.Empty;
 
         /// <summary>
-        /// The path of user profile cache, used for generating authoer and contributors.
+        /// The address of user profile cache, used for generating authoer and contributors.
+        /// It should be an absolute url
         /// </summary>
-        public readonly string UserProfileCachePath = string.Empty;
+        public readonly string UserProfileCacheUrl = string.Empty;
```

Comment:
>UserProfileCacheUrl [](start = 31, length = 19)

this config could either be an URL or a file

Reply:
OK


---
In reply to: [202250466](https://github.com/dotnet/docfx/pull/3065#discussion_r202250466) [](ancestors = 202250466)